src/hooks/internal/useMessagesInternal.ts

```diff
@@ -31,6 +31,64 @@ export const useMessagesInternal = () => {
 	// handles rcb events
 	const { callRcbEvent } = useRcbEventInternal();
 
+	/**
+	 * Simulates the streaming of a message from the bot.
+	 * 
+	 * @param message message to stream
+	 * @param streamSpeed speed to stream the message
+	 * @param useMarkup boolean indicating whether markup is used
+	 */
+	const simulateStream = useCallback(async (message: Message, streamSpeed: number, useMarkup: boolean) => {
+	// stop bot typing when simulating stream
```

Comment:
Comment is not aligned properly with the statement it belongs to, a few other instances of this below

Reply:
Pushed.